src/api/index.js

```diff
@@ -1,17 +1,19 @@
-import { version } from "../../package.json";
 import { Router } from "express";
-
-import messages from "./handlers/messages";
+import { version } from "../../package.json";
 import appointments from "./handlers/appointments";
-import replies from "./handlers/replies";
+import auth from "./handlers/auth";
 import languages from "./handlers/languages";
+import messages from "./handlers/messages";
+import replies from "./handlers/replies";
 import messageTemplates from "./handlers/templates";
 
+
 const { patchAppointments, postAppointments, getAppointments } = appointments;
```

Comment:
you can do the same destructuring in the module import itself.
`import { patchAppointments, postAppointments, getAppointments } from './handlers/appointments';`

Reply:
I think something is weird about it with the way we export from the handlers. I have not invested any time in investigating beyond it looked like things break when you change it.

Reply:
Oh true, this will only work for named exports. No big deal, we can fix the exports later if we want